pkg/common/plugin/log.go

```diff
@@ -1,8 +1,13 @@
 package plugin
 
-import "github.com/sirupsen/logrus"
+import (
+	"io/ioutil"
 
-// Log provides a plugin logger to version shim implementations.
-type Log struct {
-	logrus.FieldLogger
+	"github.com/sirupsen/logrus"
+)
+
+func NullLogger() logrus.FieldLogger {
+	logger := logrus.New()
+	logger.Out = ioutil.Discard
```

Comment:
I think we should be able to use `io.Discard` now that we are on Go 1.16.

Reply:
Truth. Mind if we make those changes all in a single PR instead of starting to mix new io/os with old ioutil in the codebase?

Reply:
Sure, that sounds fine to me.